src/components/CopyURL.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+import { transitionCss } from "../styles/styleUtils";
+import Input from "./Input";
+
+const ButtonStyle = styled.button`
```

Comment:
is this section of code also copied from Button.js too? can we use the component directly instead of doing this (like the changes you did with input)?

Reply:
the padding is a lil diff ... should I create a new props for padding?

Reply:
@christylo i'm not too familiar with styled-components so not sure what the best solution is, but i read up on some docs and it looks like there is a way to override specific properties in the Button component itself 

take a look at the first example here: https://styled-components.com/docs/api